apps/wallet-mobile/e2e/screens/prepareApp.screen.ts

```diff
@@ -0,0 +1,6 @@
+import { by, element } from 'detox'
+
+export const btn_selectLanguageEnglish = () => element(by.id('languageSelect_en-US'))
```

Comment:
Please change the naming of the constants in the file.
It should be <name><ElementType> and the name should be in camelCase

Reply:
Done